backend/worker/kafka_worker.go

```diff
@@ -106,6 +107,9 @@ func (k *KafkaBatchWorker) flush(ctx context.Context) {
 						}
 					}
 					received += len(lastMsg.PushLogs.LogRows)
+				case kafkaqueue.PushTraces:
+					traceRows = append(traceRows, lastMsg.PushTraces.TraceRows...)
+					received += len(lastMsg.PushTraces.TraceRows)
```

Comment:
Wonder if we should make a distinction between how many logs vs how many traces are received. We obviously want to batch our writes but it feels very easy for logs and traces to create unpredictability in that behavior (for example, 99% of `received` could be traces and 1% could be logs, we flush, and then we had an unperformant write for logs.

Reply:
Yeah, I think we will want to handle traces separately. Might do something similar to what @mayberryzane did in https://github.com/highlight/highlight/pull/6133/files#diff-4af9641b11571fe8e394b9583d07b44440a1f742db21dc47196319614517f95d.